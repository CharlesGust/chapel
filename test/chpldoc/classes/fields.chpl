class fullOfFields {
  var thing1;
  var thing2;
  var typedField: int;
  var alsoTyped: string;
  /* This field has a doc attached, does that work? */
  var commented;
  /* This field has a doc and an explicit type */
  var typeComment: int;
  var initialized = "bleah";
  var initializedType: int = 3;
}

record fullOfFieldsToo {
  var thing1;
  var thing2;
  var typedField: int;
  var alsoTyped: string;
  /* This field has a doc attached, does that work? */
  var commented;
  /* This field has a doc and an explicit type */
  var typeComment: int;

}