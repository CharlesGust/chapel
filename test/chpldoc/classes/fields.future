bug: type output is incorrect