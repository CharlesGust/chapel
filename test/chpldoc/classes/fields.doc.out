Module: fields
   Class: fullOfFields
      field thing1
      field thing2
      field typedField: int(64)
      field alsoTyped: string
      field commented
          This field has a doc attached, does that work? 
      field typeComment: int(64)
          This field has a doc and an explicit type 
      inherited from object

   Record: fullOfFieldsToo
      field thing1
      field thing2
      field typedField: int(64)
      field alsoTyped: string
      field commented
          This field has a doc attached, does that work? 
      field typeComment: int(64)
          This field has a doc and an explicit type 
